src/overlays/actors/ovl_Obj_Warp2block/z_obj_warp2block.h

```diff
@@ -7,12 +7,18 @@
 struct ObjWarp2block;
 
 typedef void (*ObjWarp2blockActionFunc)(struct ObjWarp2block*, GlobalContext*);
+typedef s32 (*OtherFunc)(struct ObjWarp2block*, GlobalContext*);
```

Comment:
Since this is in the header file, it should probably have a name specific to this actor.
For example:
```suggestion
typedef s32 (*ObjWarp2blockFunc168)(struct ObjWarp2block*, GlobalContext*);
```